betty/extension/webpack/assets/working/package.json

```diff
@@ -10,11 +9,14 @@
     "clean-webpack-plugin": "^4.0.0",
     "core-js": "^3.26.0",
     "css-loader": "^6.7.1",
```

Comment:
Slim this entire extension down. We need it for the requirements, but that's it.